src/ValueObject/Username.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ValueObject;
+
+use App\Exceptions\ValueObject\Username\InvalidUsernameException;
+
+final class Username
+{
+    private const CONSTRAINT = 'The username must be all alphanumeric with `_`, `-`, `.`, without spaces and be at least 5 characters long.';
+    private const CONSTRAINT_REGEX = '/^[a-zA-Z0-9\._-]{5,}$/';
```

Comment:
This forces to create an user with at least 5 characters when using command.
Also constraints like this should be handled by Symfony validator, i.e using `@Assert`